src/components/Book.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Book = book => (
```

Comment:
book should be an object right?
![image](https://user-images.githubusercontent.com/31897434/74706017-15fece00-5216-11ea-9f2e-5e707a62e4aa.png)

It could be like this 👇 
`function Book({ book }) {}`